### ISSUE TITLE
[a11y] Make hyperlinks accessible

### ISSUE THREAD

**Issue by rachelkang** [rachelkang/recipeSearch#4]
Currently, hyperlinks in labels are not accessible. They cannot be accessed when a screen reader is activated
